sites/all/modules/_custom/forms/static/nc_rename_items.js

```diff
@@ -110,25 +112,35 @@ jQuery(document).ready(function($) {
 	rename_button.on("click",function(e){
 		e.preventDefault();
 		var new_val = new_name.val();
-		//check if the new name is empty and if so alert
 
-		if(new_val == ""){
-			$('#rename_label').append('<b style="color: red">  New name cannot be empty !</b>');
+		//check if the new name is empty and if so alert
+		if(isNameEmpty(new_val)){
+			showErrorTextbox("new_item_name","Empty value");
 			return;
 		}
 
-		var orig_fol_name = original_folder.val();
+		var max_len;
 		var to_change = $("input[name='choose_item_rename']:checked").val();
+		//check if lengths are in limit
+		
+		if(to_change === "Folder")
+			max_len = max_folder_name_len;
```

Comment:
there is a way to do it using Drupal form API directly. you can look into that as well.